src/main/kotlin/com/linchpino/core/repository/InterviewRepository.kt

```diff
@@ -11,6 +11,12 @@ import org.springframework.stereotype.Repository
 
 @Repository
 interface InterviewRepository : JpaRepository<Interview, Long> {
+    @Query(
```

Comment:
This query does not need Case and you can write more simple query for it. Also use multiline string.